ocs_ci/templates/ocs-deployment/operator-openshift.yaml

```diff
@@ -216,6 +216,25 @@ spec:
         - name: ROOK_CSI_ENABLE_RBD
           value: "{{ rook_csi_enable_rbd | default('true') }}"
 
+# CSI sidecar image definition for https://jira.coreos.com/browse/RHSTOR-150
```

Comment:
@petr-balogh operator-openshift.yaml doesn't even have explicit mention of following:
**- name: ROOK_CSI_CEPH_IMAGE**

Hence,  in deployment, to be able to enforce DS image for cephCSI as well (non-sidecar) from conf/ocsci/downstream_config.yaml  , we need to add the **if** checker for "ROOK_CSI_CEPH_IMAGE" as well.
```
 "ceph_csi_image: 'quay.io/rhceph-dev/cephcsi:latest'" 
```

Reply:
Yep, as this PR was related to just sidecars images I haven't realized that once you moved to operator-openshift.yaml that we are missing this CSI Ceph image definition which I had here:  https://github.com/red-hat-storage/ocs-ci/commit/b4146356fd12e2b64228da9a33e8e1026a63d51c#diff-facdd990c9c3a9bd4151810dc4889cd4L126-L127 . Added in next commit.

Reply:
@petr maybe I missed something, but none of my CSI pods came up with the "quay.io/rhceph-dev/cephcsi:latest"

2 issues seen:

1. Even with below settings in place, ceph-agent pods came up ( not expected)

2. Nonce of the CSI pods are coming up. They are in crashloop state.

```

ery 2.0s: oc get pods -o wide -n openshift-storage                                                                             Mon Aug 12 18:58:32 2019

NAME                                          READY   STATUS             RESTARTS   AGE     IP             NODE                                         N
OMINATED NODE   READINESS GATES
csi-cephfsplugin-d2d5v                        1/2     CrashLoopBackOff   4          4m35s   10.0.138.191   ip-10-0-138-191.us-east-2.compute.internal   <
none>           <none>
csi-cephfsplugin-provisioner-0                2/3     CrashLoopBackOff   4          4m35s   10.128.2.17    ip-10-0-138-191.us-east-2.compute.internal   <
none>           <none>
csi-cephfsplugin-qn2f7                        1/2     CrashLoopBackOff   5          4m35s   10.0.172.97    ip-10-0-172-97.us-east-2.compute.internal    <
none>           <none>
csi-cephfsplugin-v664v                        1/2     CrashLoopBackOff   4          4m35s   10.0.149.187   ip-10-0-149-187.us-east-2.compute.internal   <
none>           <none>
csi-rbdplugin-5fr2q                           1/2     CrashLoopBackOff   4          4m35s   10.0.149.187   ip-10-0-149-187.us-east-2.compute.internal   <
none>           <none>
csi-rbdplugin-mzfz7                           1/2     CrashLoopBackOff   4          4m35s   10.0.138.191   ip-10-0-138-191.us-east-2.compute.internal   <
none>           <none>
csi-rbdplugin-provisioner-0                   3/4     CrashLoopBackOff   4          4m35s   10.131.0.13    ip-10-0-172-97.us-east-2.compute.internal    <
none>           <none>
csi-rbdplugin-w7622                           1/2     CrashLoopBackOff   5          4m35s   10.0.172.97    ip-10-0-172-97.us-east-2.compute.internal    <
none>           <none>

```

```
---
ENV_DATA:
  ceph_image: 'quay.io/rhceph-dev/rhceph-4.0-rhel-8:latest'
  ceph_csi_image: 'quay.io/rhceph-dev/cephcsi:latest'
  rook_image: 'quay.io/rhceph-dev/rook:latest'
  # DS CSI sidecars images mentioned in https://jira.coreos.com/browse/RHSTOR-150
  rook_csi_registrar_image: 'quay.io/openshift/origin-csi-node-driver-registrar:4.2'
  rook_csi_provisioner_image: 'quay.io/openshift/origin-csi-external-provisioner:4.2'
  rook_csi_snapshotter_image: 'quay.io/openshift/origin-csi-external-snapshotter:4.2'
  rook_csi_attacher_image: 'quay.io/openshift/origin-csi-external-attacher:4.2'
  cluster_namespace: 'openshift-storage'
REPORTING:
  # Upstream: 'US' or Downstream: 'DS', used only for reporting (Test Run Name)
  us_ds: 'DS'
```

Reply:
I ran the verification job with this change, so let's see how it will go with jenkins run. If it will fail as you said I will need to start investigating the issue once I will have the environment.